package.json

```diff
@@ -89,12 +89,12 @@
       "packages/fluentui/*"
     ],
     "nohoist": [
-      "packages/web-components/typescript",
-      "packages/web-components/typescript/**",
-      "packages/web-components/ts-loader",
-      "packages/web-components/ts-loader/**",
+      "packages/web-components/@microsoft/api-extractor",
```

Comment:
If we leave web-components on a different TS version, it also needs its own API Extractor version (which shouldn't be hoisted to avoid interference from API Extractor's additional separate TS version)